BUILD.md

```diff
@@ -16,7 +16,7 @@
 
 1. Python >= 3.7 (3.6 may work, 3.5 and earlier is not supported)
 1. CMake >= 3.10.2
-1. C++ >= c++11 compiler. See platform-specific sections below for supported compiler versions.
+1. C++ >= c++17 compiler. See platform-specific sections below for supported compiler versions.
```

Comment:
Line 482 also has a mention of C++11

"This repository has been built and tested on the two most recent Ubuntu LTS
versions. Currently, the oldest supported version is Ubuntu 16.04, meaning
that the minimum officially supported C++11 compiler version is GCC 5.4.0,
although earlier versions may work. It should be straightforward to adapt this
repository to other Linux distributions."